src/router.ts

```diff
@@ -9,11 +10,15 @@ import * as user from './requestHandlers/user';
 export default (app: Express) => {
     app.get('/', help);
     app.get('/help', help);
-
+    app.get('/getUserByID', user.getUserByID);
     app.post('/createUser', user.createUser);
 
     // this will be changed to GET and use the authentication header instead of body
     app.post('/authenticate', user.authenticate);
     app.post('/changePassword', user.changePassword);
     app.post('/changeDisplayName', user.changeDisplayName);
+
+    app.post('/createPanelSet', panelSet.createPanelSet);
+    app.get('/getPanelSetByID', panelSet.getPanelSetByID);
+    app.get('/getAllPanelSetsFromUser', panelSet.getAllPanelSetFromUser);
```

Comment:
Function name does not match command (`/getAllPanelSets...` vs `.getAllPanelSet...`)